### DISCUSSION TITLE
Navigation with expanding submenus

### DISCUSSION THREAD

**Discussion by jasonbouffard** [beercss/beercss#317] [Ideas]
Does anyone have any ideas re expanding submenus?

<img width="230" alt="Screenshot 2024-06-11 at 9 19 55 AM" src="https://github.com/beercss/beercss/assets/181576/5e9399f5-36b9-4593-a465-4551b086d1ef">

```
<nav class="left drawer small-elevate">
  <header class="padding">
    <img src="/img.png" class="">
  </header>
  <details>
    <summary class="none">
      <article class="no-elevate">
        <nav>
          <i>shopping_cart</i>
          <div>Items</div>
        </nav>
      </article>
    </summary>
    <nav class="drawer">
      <a><div>Item list</div></a>
      <a><div>Item types</div></a>
      <a><div>Discounts</div></a>
    </nav>
  </details>
```
That was my first try. Just curious.

https://codepen.io/Jason-Bouffard/pen/VwOMywG

**Comment by leonardorafael** [comment]
Hi,

Currently, we don't have an expandable items for navigation drawer, but you can use the same logic of lists here.
Basically the trick is to add the `row` and `wave` helpers to all items, combined with `<details`> and `<summary>` elements.
And this is a pure CSS solution.

https://codepen.io/leo-bnu/pen/Baewrgg

This should work inside your navigation drawer:

```html
<details>
  <summary class="none">
    <a class="row wave small-round">
      <i>shopping_cart</i>
      <div>Items</div>
    </a>
  </summary>
  <a class="row wave small-round">Item list</a>
  <a class="row wave small-round">Item types</a>
  <details>
    <summary class="none">
      <a class="row wave small-round">Discounts</a>
    </summary>
    <a class="row wave small-round">More discounts</a>
  </details>
</details>
```

Hope this helps @jasonbouffard

**Reply by jasonbouffard** [reply]
This is great! Thank you. You could add this example to your setup html project example where you have the "More" nav. https://codepen.io/leo-bnu/pen/yLKLPxj
